tests/relay/test_fields.py

```diff
@@ -1175,3 +1177,24 @@ def test_query_connection_total_count_sql_queries(
     assert result.data == {
         query_attr: {"totalCount": 5},
     }
+
+
+@pytest.mark.skip(
+    reason="TODO: Fix issue => https://github.com/strawberry-graphql/strawberry-django/issues/535"
+)
+def test_query_lazy_connection_with_filters(db):
+    color = Color.objects.create(name="Magenta")
+    Fruit.objects.create(name="Apple", color=color)
+    query = """
+    query TestQuery {
+        colorWithLazyFruits(filters: {}) {
```

Comment:
I'm checking here and this is not supposed to have `filters`. What is supposed to have filters is `fruits`, and in my tests it seems to have